index.html

```diff
@@ -70,5 +70,32 @@
         </div>
       </div>
     </div>
+    <div class="about">
+      <div class="meet_team">Meet your Pharmacy Team</div>
+      <div class="meet1">
```

Comment:
I'd suggest to call all of the `meet1`, `meet2`, and `meet3` things just like `meet` since the styles should be the same for all of them so in the CSS file you don't have to list out each one and you can just do `.meet`.

Reply:
Same comment for `meet_pic`, `meet_person`, `title`, and `about`.